code-challenges.test.js

```diff
@@ -38,6 +98,13 @@ const studyMinutesWeek1 = {
   friday: 15,
   saturday: 60
 }
+
+describe("studyMinutesWeek1", () => {
+  it("returns an array of the object's values sorted from least to greatest", () => {
+    expect(studyMinutesWeek1()).toEqual([15, 15, 20, 30, 30, 60, 90])
+  })
+})
```

Comment:
These tests don't have arguments in the function call. You also did the same thing here with naming the function the same as the variables given.